diff --git a/node_modules/@stencil/sass/dist/index.d.ts b/node_modules/@stencil/sass/dist/index.d.ts
index bf0ec3d..ddb44a8 100644
--- a/node_modules/@stencil/sass/dist/index.d.ts
+++ b/node_modules/@stencil/sass/dist/index.d.ts
@@ -1,2 +1,4 @@
 import * as d from './declarations';
+import Sass from 'sass';
 export declare function sass(opts?: d.PluginOptions): d.Plugin;
+export declare const instance: typeof Sass;
diff --git a/node_modules/@stencil/sass/dist/index.js b/node_modules/@stencil/sass/dist/index.js
index 9d46071..3b4c7ca 100644
--- a/node_modules/@stencil/sass/dist/index.js
+++ b/node_modules/@stencil/sass/dist/index.js
@@ -377,3 +377,4 @@ function sass(opts = {}) {
 }
 
 exports.sass = sass;
+exports.instance = e;
