diff --git a/node_modules/@stencil/core/internal/client/index.js b/node_modules/@stencil/core/internal/client/index.js
index daa300e..98ec1b2 100644
--- a/node_modules/@stencil/core/internal/client/index.js
+++ b/node_modules/@stencil/core/internal/client/index.js
@@ -2431,7 +2431,7 @@ const bootstrapLazy = (lazyBundles, options = {}) => {
                     appLoadFallback = null;
                 }
                 if (isBootstrapping) {
-                    // connectedCallback will be processed once all components have been registered
+                    // connectedCallback will be Processed once all components have been registered
                     deferredConnectedCallbacks.push(this);
                 }
                 else {
